libs/cardano-ledger-api/src/Cardano/Ledger/Api.hs

```diff
@@ -0,0 +1,26 @@
+-- | This module provides a library interface for working with types that will allow a user to
+-- interact with Cardano.
+--
+-- It is intended to be the complete API covering everything but without exposing constructors that
+-- reveal any lower level types.
+--
+-- In the interest of simplicity it glosses over some details of the system.
+-- Most simple tools should be able to work just using this interface, however you can go deeper and
```

Comment:
This interface should be as powerful as using the underlying internals, but in a simpler way.
```suggestion
-- Most tools should be able to work just using this interface, however you can go deeper and
```